### PR TITLE
Delete imageUrl field from category. Fix tests

### PR DESCRIPTION
**Pull request by Bozhokmaria**

### CODE REVIEW

File: src/test/resources/sql/clean-up.sql [hunk 1]
```diff
@@ -1,9 +1,11 @@
-SET FOREIGN_KEY_CHECKS = 0;
-TRUNCATE TABLE location;
-TRUNCATE TABLE vendor;
-TRUNCATE TABLE category;
-TRUNCATE TABLE discount;
-TRUNCATE TABLE role;
-TRUNCATE TABLE tag;
-TRUNCATE TABLE "user";
-SET FOREIGN_KEY_CHECKS = 1;
+SET
```
Comment (ahlukhouski2) [inline src/test/resources/sql/clean-up.sql]: let's write in one line

File: src/test/resources/sql/clean-up.sql [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+    FOREIGN_KEY_CHECKS = 0;
+TRUNCATE TABLE location RESTART IDENTITY;
+TRUNCATE TABLE vendor RESTART IDENTITY;
+TRUNCATE TABLE category RESTART IDENTITY;
+TRUNCATE TABLE discount RESTART IDENTITY;
+TRUNCATE TABLE role RESTART IDENTITY;
+TRUNCATE TABLE tag RESTART IDENTITY;
+TRUNCATE TABLE "user" RESTART IDENTITY;
+SET
```
Comment (ahlukhouski2) [inline src/test/resources/sql/clean-up.sql]: let's write in one line

Comment (Bozhokmaria) [reply src/test/resources/sql/clean-up.sql]: +

File: src/test/resources/sql/clean-up.sql (hunk 1 above)
Comment (Bozhokmaria) [reply src/test/resources/sql/clean-up.sql]: +

**Review by dmitriychernov7 — APPROVED** [review]

**Pull request merged**